td/publishing/models.py

```diff
@@ -260,31 +283,50 @@ class LicenseAgreement(models.Model):
 
 class Chapter(models.Model):
     resource_type = models.ForeignKey(OfficialResourceType)
+    publish_request = models.ForeignKey(PublishRequest, default=None, null=True)
+    created = models.DateTimeField(default=timezone.now)
     language = models.ForeignKey(Language)
     number = models.IntegerField()
-    ref = models.TextField()
+    ref = models.TextField(default=None, null=True)
```

Comment:
For `TextField` and `CharField` setting to just `blank=True` is preferred, so as not not mix empty string and `null` as having two *falesy` values to contend with.  `blank=True` will use an empty string by default on these fields.